documentation/api/query/v4/reports.markdown

```diff
@@ -70,6 +70,15 @@ The following fields are allowed as filter criteria and are returned in all resp
 
 * `transaction_uuid` (string): a string used to identify a Puppet run.
 
+* `catalog_uuid` (string): a string used to tie a catalog to a report to the
+  catalog used from that Puppet run.
+
+* `code_id` (string): a string used to tie a catalog to the Puppet code which generated the catalog.
+
+* `cached_catalog_status` (string): a string used to identify whether the Puppet
```

Comment:
I can't find an authoritative document to point this at. It would seem to me that the report processor format document might contain this.